lib/upgrade_html.js

```diff
@@ -589,4 +595,92 @@ function insertHtmlCommentBefore($, elem, commentLines) {
   $(elem).before(commentText);
 }
 
+function upgradeGlobalCss($) {
+  var cssRules = {};
+  $(upgradeCss.needsUpgradeQuerySelector).each(function(_ignored, elemNeedsUpgrade) {
+    _.extend(
+        cssRules,
+        upgradeCss.getCssRulesNeededToUpgrade($(elemNeedsUpgrade)));
+  });
+  if (!_.isEmpty(cssRules)) {
+    var style = $('<style>');
+    style.attr('is', 'custom-style');
+    style.text(getCssTextGivenRules(cssRules));
+    $('head').append('  ');
+    $('head').append(style);
+    $('head').append('\n');
+    addHtmlImport($, 'iron-flex-layout/iron-flex-layout.html');
+  }
+}
+
+function upgradeElementCss($, polymerElement, domModule, templateElem) {
+  var cssRules = upgradeCss.getCssRulesNeededToUpgrade($(polymerElement));
+  for (var selector in cssRules) {
+    cssRules[':host' + selector] = cssRules[selector];
+    delete cssRules[selector];
+  }
+  var elemsNeedingUpgrade = recursivelyMatchInsideTemplates(
+      $, $(templateElem), upgradeCss.needsUpgradeQuerySelector);
+  elemsNeedingUpgrade.forEach(function(elemNeedsUpgrade) {
+    _.extend(
+        cssRules,
+        upgradeCss.getCssRulesNeededToUpgrade($(elemNeedsUpgrade))
+    );
+  });
+  if (!_.isEmpty(cssRules)) {
+    var style = $('<style>');
+    style.text(getCssTextGivenRules(cssRules));
```

Comment:
it'd be nice to drop a comment into this `<style>` explaining that the `@apply()` should ideally be moved into other style rules.